### ISSUE TITLE
.config/awesome/pomodoro/impl.lua:30: attempt to index field 'widget' (a nil value)

### ISSUE THREAD

**Issue by mkonecny** [nikolavp/awesome-pomodoro#26]
Is this widget for Awesome 3.5 only? Currently running 3.4 (latest on Debian Jessie 8.4), and seeing this error.

**Comment by nikolavp** [comment]
Yes, sorry. It seems that at some point I broke support for 3.4 :( It seems the problem is that widget is no longer a constructor on which you have to provide a type but different objects under the wibox module.

Can you provide the patch that enables the 3.4 support? Otherwise I will have to find the time to patch this at some point.
